sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_session_receiver_async.py

```diff
@@ -134,7 +133,8 @@ def from_connection_string(
                 :caption: Create a new instance of the ServiceBusReceiver from connection string.
 
         """
-        return super(ServiceBusSessionReceiver, cls).from_connection_string(conn_str, **kwargs)
+        constructor_args = super(ServiceBusSessionReceiver, cls)._from_connection_string(conn_str, **kwargs)
```

Comment:
I _believe_ it would have done the right behavior even in the single-call form, since it was using "cls" to initialize the object regardless; but if we want to decouple it further, let's at least name this inner function (_from_connection_string) something more useful than just a magic hidden callable of the same structure.  Perhaps "_get_constructor_args_from_connection_string" (Yes, verbose, but very clear ><)

Reply:
You're right. The single-call form is right. It's a false alarm of mypy. I'll change it back and suppress that mypy error. The original way is neat.